Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/CameraFollow.cs

```diff
@@ -4,22 +4,27 @@ namespace Unity.MLAgentsExamples
 {
     public class CameraFollow : MonoBehaviour
     {
-        public Transform target;
-        Vector3 m_Offset;
+        [Tooltip("The target to follow")] public Transform target;
+
+        [Tooltip("The time it takes to move to the new position")]
+        public float smoothingTime; //The time it takes to move to the new position
+
+        private Vector3 m_Offset;
+        private Vector3 m_CamVelocity; //Camera's velocity (used by SmoothDamp)
 
         // Use this for initialization
         void Start()
         {
             m_Offset = gameObject.transform.position - target.position;
         }
 
-        // Update is called once per frame
-        void Update()
+        void FixedUpdate()
         {
-            // gameObject.transform.position = target.position + offset;
             var newPosition = new Vector3(target.position.x + m_Offset.x, transform.position.y,
                 target.position.z + m_Offset.z);
-            gameObject.transform.position = newPosition;
+
+            gameObject.transform.position =
+                Vector3.SmoothDamp(transform.position, newPosition, ref m_CamVelocity, smoothingTime);
```

Comment:
I think you need to add the argument deltaTime=Time.fixedDeltaTime since this is now in the fixed update.

Reply:
changed. i didn't even know that was an option.